### ISSUE TITLE
nixos/wakapi: StateDirectory is not set

### ISSUE THREAD

**Issue by bbigras** [NixOS/nixpkgs#347393]
### Describe the bug

I'm not 100% sure but shouldn't `StateDirectory` be set to `wakapi` so that `/var/lib/wakapi` is created and accessible from the service?

### Steps To Reproduce
1. ...
2. ...
3. ...

### Notify maintainers

@isabelroses 

### Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.1, NixOS, 24.11 (Vicuna), 24.11.20241004.bc947f5`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.90.0-lixpre20240527-0b91a4b`
 - nixpkgs: `/run/current-system/nixpkgs`

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by Princemachiavelli** [comment]
Yes, I would assume so. The module sets `StateDirectoryMode` so I assume the service uses it's `StateDirectory`.

**Comment by bbigras** [comment]
I set StateDirectory on my side and it seems to work. I can make a PR.
